circle.yml

```diff
@@ -4,9 +4,12 @@ machine:
   environment:
     BUILD_DIR: $HOME/$CIRCLE_PROJECT_REPONAME
     DB_NAME: "circle_test"
-    DB_USER: "ubuntu"
+    DB_USER: "root"
     DB_PASS: ""
     DB_HOST: "127.0.0.1"
+    DB_SOURCE_DUMP: "$BUILD_DIR/.data/db.dist.sql"
+    DB_CONTAINER_TAR: "$BUILD_DIR/.data/db-container.tar"
```

Comment:
please make this variable inline as the end user does not need to change it to make things work

Reply:
I think it is useful to retain this environment variable as it is a parameter to the container build script. 

Even though the end-user doesn't have to change it, it means we have consistent paths between `circle.yml` and the bash script.